<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>368</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>451</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>451</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Decoding">
      <attribute name="title">
       <string>Decoding</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>401</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Source</string>
       </property>
       <widget class="QComboBox" name="comboBoxsoundcard">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>381</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>401</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Soundcard out</string>
       </property>
       <widget class="QComboBox" name="comboBoxsoundcard_out">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>381</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QCheckBox" name="checkBox_tracking">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>200</y>
         <width>101</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Tracking</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_cma">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>240</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>CMA</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_fec_type">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>63</width>
         <height>21</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>soft</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hard</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>202</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Decoder type</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_sdr">
      <attribute name="title">
       <string>RTL SDR</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>401</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>RTL SDR Device</string>
       </property>
       <widget class="QComboBox" name="comboRTLDevices">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>381</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>401</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QCheckBox" name="checkBox_rtl_afc">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>12</y>
          <width>91</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>AFC</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_rtl_agc">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>42</y>
          <width>91</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>AGC</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_rtl_gain">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="suffix">
         <string> dB</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>50.000000000000000</double>
        </property>
        <property name="value">
         <double>50.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>13</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Subcarrier Offset</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>42</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Gain</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_rtl_subcarrier_freq_offset">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="suffix">
         <string> kHz</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_rtl_filter_selection">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>87</width>
          <height>22</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>100kHz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>120kHz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>150kHz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200kHz</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Filter selection</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
